e2e/framework/clients/rook_test_client.go

```diff
@@ -0,0 +1,101 @@
+package clients
+
+import (
+	"fmt"
+	"github.com/rook/rook/e2e/framework/contracts"
+	"github.com/rook/rook/e2e/framework/enums"
+	"github.com/rook/rook/e2e/framework/transport"
+	"github.com/rook/rook/pkg/model"
+)
+
+var (
+	VERSION_CMD = []string{"rook", "version"}
+)
+
+type RookTestClient struct {
```

Comment:
I like having clear distinction between test client and actual client, so others using it don't import the wrong version(especially IDEs which do auto-imports).